portfolio/src/main/java/com/google/sps/servlets/FunFactsServlet.java

```diff
@@ -0,0 +1,40 @@
+package com.google.sps.servlets;
+
+import java.util.ArrayList;
+import java.io.IOException;
+import com.google.gson.Gson;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+/** Handles requests sent to the /hello URL. Try running a server and navigating to /hello! */
+@WebServlet("/funfacts")
+public class FunFactsServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+    ArrayList<String> funFacts = new ArrayList<String>();
```

Comment:
Tip: You can use shorthand syntax to avoid having many `put` statements within your function: 
```java
String[] funFacts = { "..", "..." };
```